geode-core/src/test/java/org/apache/geode/test/dunit/cache/internal/JUnit4CacheTestCase.java

```diff
@@ -368,6 +368,7 @@ public void postTearDownCacheTestCase() throws Exception {
   private static synchronized void remoteTearDown() {
     try {
       DistributionMessageObserver.setInstance(null);
+      waitForNoRebalancing();
```

Comment:
Looks like this is test specific...how about having this in "CacheXmlTestCase" instead here...